File: pair_impair.js
let nombre, reste;
const readline = require('readline');

// Créer une interface de lecture pour lire depuis le clavier
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

// Poser une question à l'utilisateur
rl.question('Veuillez saisir un nombre : ', (answer) => {
  nombre= answer;
  reste = nombre%2;
  if (reste ==0){
    console.log("le nombre "+nombre+" est pair"); 
  }else{
    console.log("le nombre "+nombre+" est impair");
  }
  rl.close();
}); 

File: pair_impair.java
import java.util.Scanner;

public class pair_impair {

	public static void main(String[] args) {
		
		
		//Déclaration des variables
		double nombre, reste; 
		
		// Enregistrement des valeurs tapées par l'utilisateur
		Scanner clavier = new Scanner(System.in); 
		System.out.print("veuillez saisie une nombre: ");
		nombre = clavier.nextDouble();
		reste = nombre%2;
		
		if(reste==0) {
			System.out.print("le nombre "+nombre+" est pair");
		}else {
			System.out.print("le nombre "+nombre+" est impair");
		}
		
		clavier.close(); 
			
	}

}
